### PR TITLE
chore(FDS-508): [Rollup] - bump rollup to v2.67.3 and update ecosystem

### PR DESCRIPTION
**Pull request by lordpixel**
Bump rollup to `v2.67.3` and update ecosystem.

## PR Author Checklist

- [x] PR title adheres to [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#specification)
- [x] I have finished reviewing the contents of this PR for accuracy
- [x] If needed, I have included a [Changeset](https://github.com/changesets/changesets) and it follows [Semantic Versioning principles](https://semver.org/)
- [x] I have completed all of the above and have enabled `auto-merge` for this PR

### CODE REVIEW

**Comment by lordpixel** [comment]
> Messages
> 📖	Dangerfile.js does not run when the current branch is chore/FDS-511-update-rollup
> Generated by 🚫 [dangerJS](https://danger.systems/js) against [57a2254](https://github.com/Espressive/cascara/commit/57a2254bc5e414d7f5a261d6b1066f4e319521a4)

The above message should not appear, we need to fix the Danger.js config so it doesn't skip these kind of flow.

**Comment by rociobaena** [comment]
> > Messages
> > 📖	Dangerfile.js does not run when the current branch is chore/FDS-511-update-rollup
> > Generated by 🚫 [dangerJS](https://danger.systems/js) against [57a2254](https://github.com/Espressive/cascara/commit/57a2254bc5e414d7f5a261d6b1066f4e319521a4)
> 
> The above message should not appear, we need to fix the Danger.js config so it doesn't skip these kind of flow.

Hi Manu. I'll start working on this fix.

**Comment by rociobaena** [comment]
> > > Messages
> > > 📖	Dangerfile.js does not run when the current branch is chore/FDS-511-update-rollup
> > > Generated by 🚫 [dangerJS](https://danger.systems/js) against [57a2254](https://github.com/Espressive/cascara/commit/57a2254bc5e414d7f5a261d6b1066f4e319521a4)
> > 
> > 
> > The above message should not appear, we need to fix the Danger.js config so it doesn't skip these kind of flow.
> 
> Hi Manu. I'll start working on this fix.

Hi Manu. The Danger message is correct. The idea is that Danger should only run when the base is different from develop and main and the target is develop or main. But we can change that. We can run danger in every branch. Let's talk about it, and I'll work in our resolution.

**Review by rociobaena — APPROVED** [review]

File: ADR/rollup.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Rollup
```
Comment (odiseo42) [inline ADR/rollup.md:1]: Fantastic ADR!

**Review by odiseo42 — APPROVED** [review]
LGTM

**Pull request merged**